test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -711,6 +711,49 @@ abc bcd
 
 
 
+"@ }
+        ) {
+            param($obj, $expectedTable)
+            $output = $obj | Format-Table | Out-String
+            $output.Replace("`r","").Replace(" ",".").Replace("`n","^") | Should -BeExactly $expectedTable.Replace("`r","").Replace(" ",".").Replace("`n","^")
+        }
+
+        It "Should render header correctly where header is shorter than column width with justification: <variation>" -TestCases @(
+            @{ variation = "left/right"; obj = [PSCustomObject]@{a="abc";b=123}; expectedTable = @"
+
+a     b
+-     -
+abc 123
+
+
+
+"@ },
+            @{ variation = "left/left"; obj = [PSCustomObject]@{a="abc";b="abc"}; expectedTable = @"
+
+a   b
+-   -
+abc abc
+
+
+
+"@ },
+            @{ variation = "right/left"; obj = [PSCustomObject]@{a=123;b="abc"}; expectedTable = @"
+
+  a b
+  - -
+123 abc
```

Comment:
Interesting, 
1. should we justify header fields? Maybe always have left/center/right justify?
2. should we pad "-" to length of values not headers?

```
a   b
--- ---
123 abc
```

Reply:
Maybe, but that's an enhancement.  I would suggest opening a new issue to discuss that.  This PR makes this consistent with Windows PowerShell existing behavior.

Reply:
Does this mean that this issue only happens on PS Core?

Reply:
@anmenaga yes, this is a regression in PSCore6.1